canard.c

```diff
@@ -151,29 +151,20 @@ int16_t canardBroadcast(CanardInstance* ins,
     uint32_t can_id = 0;
     uint16_t crc = 0xFFFFU;
 
-    if (canardGetLocalNodeID(ins) == 0)
+    can_id = ((uint32_t) priority << 26U) | ((uint32_t) data_type_id << 8U) | ((uint32_t) canardGetLocalNodeID(ins) << 1U);
```

Comment:
If the local node ID is 0xFF, the shift operation overwrites the least significant bit of the subject ID.